docs/reference/changelog.rst

```diff
@@ -2,6 +2,18 @@
 Changelog
 *********
 
+2.2.1 (2024-MM-DD)
+------------------
+
+Bug fixes:
+
+- Fix how extras and groups are parsed for the
+  :ref:`uv plugin<craft_parts_uv_plugin>`
+
+Other improvements:
+
+- Reorganize tests for Python-based plugins to avoid repeating tests
```

Comment:
I know we had a team bikeshedding moment a month or two ago where we added test updates to the changelog but I'm of the opinion this can be omitted from the changelog because users of the library don't need to know about internal testing.

Reply:
Agreed - Any changes to the tests directory can be skipped in the changelog (but the attached bugfix is good to keep IMO)